src/content/protocols/sky.md

```diff
@@ -0,0 +1,381 @@
+---
+protocol: "Sky"
+website: "https://sky.money/"
+x: "https://x.com/SkyEcosystem"
+github: ["https://github.com/makerdao"]
+defillama_slug: ["sky-lending"]
+chain: "Ethereum"
+stage: 0
+reasons: []
+risks: ["L", "H", "M", "H", "L"]
+author: ["mmilien_"]
+submission_date: "2025-03-11"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Sky is a stablecoin protocol allowing users to mint its USDS stablecoin through _Collateralized Debt Positions_ with a variety of crypto collateral assets. Sky is built on the Maker protocol, by the MakerDAO, with the objective to replace the DAI stablecoin and allows for the 1:1 conversion between DAI and USDS. USDS can further be staked for the yield-bearing sUSDS token. Various levels of control over the Sky protocol are exercised through an onchain governance system.
+
+# Overview
+
+## Chain
+
+This review focuses on the Ethereum deployment of Sky, currently the only chain the Sky protocol is deployed on. As part of Sky's _End Game_ plan the protocol is gradually deploying on other chains and aims to launch its own chain in the future.
+
+> Chain score: Low
+
+## Upgradeability
+
+`USDS` and `sUSDS` are upgradeable contracts through governance proposals. Updating those contracts could change the entire logic of those tokens and may incur loss of funds for users.
+
+Many parameters in the Sky Protocol can be changed through governance proposals and may also incur loss of funds, loss of unclaimed yield, as well as material changes in expected performances. Example actions are forced liquidations, creating unbacked debt, or pausing the contracts.
```

Comment:
Expand the "Examples" a bit and give main areas where updates can result in:
1. loss of funds
2. loss of unclaimed yield
3. otherwise materially impact protocol performance